packages/codegen/generate.server.utils.js

```diff
@@ -233,6 +234,27 @@ function generateServerUtils (gql) {
         })
     }
 
+    async function updateMany (context, data, extraAttrs = {}, options = {}) {
+        if (!context) throw new Error('no context')
+        if (!data) throw new Error('no data')
+        _checkOptions(options)
+
+        const chunks = splitArray(data, MAX_CHUNK_SIZE)
+        let result = []
+
+        for (const chunk of chunks) {
```

Comment:
Are you sure, that chunks should go here and not to mutations itself?
And why number 50 is chosen? Did you test limits of it?

My comment was about the fact that we hardly ever used mass mutations before. That's why I asked, have you tested the limits? Can updateMany handle 100/1000/10000 objects per request?

Reply:
Changed max chunk size to 100 -- tested how many update operations updateMany can handle and it depends on the number of fields with type Relationship (for 5 fields it can do 200 operations, for 13 - 168 operations). So decided to make it smaller just in case.